src/web/mjs/connectors/templates/FlatManga.mjs

```diff
@@ -85,7 +85,10 @@ export default class FlatManga extends Connector {
                 try {
                     element.dataset.pagespeedLazySrc = atob(element.dataset.pagespeedLazySrc);
                 } catch(_) { /* ignore */ }
-                return this.getAbsolutePath(element.dataset.src || element.dataset.original || element.dataset.pagespeedLazySrc || element, request.url);
+                try {
```

Comment:
You can simply fuse both try/catch into a single one

Reply:
Hmmm, could be an issue as it would short-circuit by triggering the ignore error....
Isn't there a way to find out the race condition instead of trying them all ?

Reply:
idk i just did more of what it was already doing
i assumed it was like that for a reason